src/sidebar/test/bootstrap.js

```diff
@@ -1,6 +1,22 @@
 // Expose the sinon assertions.
 sinon.assert.expose(assert, { prefix: null });
 
+assert.rejects = async (asyncFn, error) => {
+  try {
+    await asyncFn;
+    const error = new Error();
+    error.name = 'ErrorNotCaught';
+    throw error;
+  } catch (e) {
+    if (e.name === 'ErrorNotCaught') {
+      throw new Error(
+        'Expected to catch the rejected promise but it was not thrown'
+      );
+    }
+    assert.equal(error, e.message);
```

Comment:
From experience with using `assert.throws`, I think it would be useful to allow the use of partial matching by passing a regex as the value of `error`.

Reply:
good point, I think I named that before I sorted out how I wanted the param to look.